doc/rtd/topics/analyze.rst

```diff
@@ -292,10 +313,23 @@ commands below will gather the UserspaceTimestamp and InactiveExitTimestamp:
 
 .. code-block:: shell-session
 
-  $ systemctl show -p UserspaceTimestampMonotonic
-  UserspaceTimestampMonotonic=989279
-  $ systemctl show cloud-init-local -p InactiveExitTimestampMonotonic
-  InactiveExitTimestampMonotonic=4493126
+   $ systemctl show -p UserspaceTimestampMonotonic
+
+Example output:
+
+.. code-block::
+
+   UserspaceTimestampMonotonic=989279
+
```

Comment:
This now looks cluttered:

![image](https://user-images.githubusercontent.com/16310367/206529381-d6056de9-e48e-4a32-a1c3-613a398be7a7.png)

Maybe we could split up the line from before: "will gather the UserspaceTimestamp and InactiveExitTimestamp:" into two sentances so that one can go between the example and the command?

Reply:
Yes, I think that would improve the overall look of it - I'll do that!